src/Utils/relayHelper.ts

```diff
@@ -0,0 +1,4 @@
+export type CleanRelayFragment<T> = Omit<
+  T,
+  "$refType" | " $fragmentRefs" | " $fragmentSpreads" | " $fragmentType"
+>
```

Comment:
**Suggestion:** Should we move this to [typeSupport](https://github.com/artsy/force/blob/main/src/Utils/typeSupport.ts)?

Reply:
Great idea! I've moved the helper.